synthesizer/event.cpp

```diff
@@ -370,5 +370,45 @@ void EventList::insert(const Event& e)
             }
       append(e);
       }
-}
 
+//---------------------------------------------------------
+//   class EventMap::fixupMIDI
+//---------------------------------------------------------
+
+void EventMap::fixupMIDI()
+      {
+      unsigned short nowPlaying[_highestChannel + 1][128 /* notes */];
```

Comment:
And using memset() is way to much C style

Reply:
The error message @lasconic cited "Variable sized object may not be initialized" is extremely clear.

VLAs have been supported since C99, and I extremely doubt C++11 does not support them. AIUI clang just fails to initialise them.

Reply:
I see now. I feel almost physical allergy to all this ancient stuff.